src/tests/eosjs-api.test.ts

```diff
@@ -89,13 +89,13 @@ const serializedActions = [
     {
         account: 'testeostoken',
         authorization: [{ actor: 'thegazelle', permission: 'active' }],
-        data: "00808A517DC354CB6012F557656CA4BA102700000000000004454F530000000014466F72206120736563757265206675747572652E", // tslint:disable-line
+        data: '00808A517DC354CB6012F557656CA4BA102700000000000004454F530000000014466F72206120736563757265206675747572652E', // tslint:disable-line
```

Comment:
Do these `tslint:disable-line` do anything anymore?

Reply:
Nope, thanks for reminding me!